.yarnrc.yml

```diff
@@ -0,0 +1,16 @@
+compressionLevel: 0
```

Comment:
I think this and `enableGlobalCache` are default settings. Can we remove them from the config to make them more concise?